pkgs/applications/printing/pappl/default.nix

```
@@ -63,9 +57,13 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "C-based framework/library for developing CUPS Printer Applications";
-    homepage = "https://github.com/michaelrsweet/pappl";
+    homepage = "https://www.msweet.org/pappl";
+    changelog = [
+      "https://github.com/michaelrsweet/pappl/releases/tag/v${version}"
+      "https://www.msweet.org/blog/2022-12-30-pappl-1.3.1.html"
+    ];
```

```suggestion
    changelog = "https://github.com/michaelrsweet/pappl/releases/tag/v${version}";
```
For the next release this won't be correct anymore. Also I am not sure if changelog can even be a list.

---

`meta.changelog` can be a list: https://nixos.org/manual/nixpkgs/stable/#var-meta-changelog

But I see your point, I'll remove the second entry

---

resolved in rebased commits